node/src/Pos/Block/Logic/VAR.hs

```diff
@@ -217,7 +217,7 @@ applyBlocks calculateLrc pModifier blunds = do
 
 -- | Rollbacks blocks. Head must be the current tip.
 rollbackBlocks
-    :: (BlockLrcMode ssc ctx m)
+    :: (MonadBlockApply ssc ctx m)
```

Comment:
So this drops the `LrcModeFull` class constraint that was apparently unused. Fine.